### PR TITLE
Update aspnet50/aspnetcore50 => dnx50/dnxcore50.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
Prepping to be merged once ready.

### CODE REVIEW

File: src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ private static RegisterDelegate ResolveRegisterDelegate()
                 return token.Register(callback, state);
             };
 
-#if NETFX_CORE || PORTABLE || ASPNETCORE50
+#if NETFX_CORE || PORTABLE || DNXCORE50
```
Comment (NTaylorMullen) [inline src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs]: Seeing this Portable, probably missed something, please take a look.

Comment (muratg) [inline src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs]: Seems OK to me.  If this was working before, should be fine now too.  /cc @BrennanConroy

**Comment by muratg** [comment]
:shipit:

File: src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs (hunk 1 above)
Comment (BrennanConroy) [inline src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs]: I'm curious to know what this portable is @davidfowl

Comment (davidfowl) [inline src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs]: This code used to be shared with the client

Comment (moozzyk) [inline src/Microsoft.AspNet.SignalR.Server/Infrastructure/CancellationTokenExtensions.cs]: I think @halter73 tried to get rid of these in 670e6d30fe but might have missed this one..

**Pull request merged**